eng/code-quality-reports/src/test/resources/com/azure/tools/checkstyle/checks/GoodLoggerCheck/ExternalLoggerLibrary.java

```diff
@@ -0,0 +1,6 @@
+import com.azure.core.util.logging.ClientLogger;
+// invalid external logger library
+import org.apache.logging.log4j;
+import org.slf4j;
+import java.util.logging;
+
```

Comment:
You should at least make this a valid class file.

Reply:
Trying to without the valid java class file to remain self this is not the java file but a text file that fake the java file to create the Abstract Syntax Tree.
But I will add the class to make a valid java class.